subprojects/code-quality/src/main/groovy/org/gradle/api/plugins/quality/CodeNarc.java

```diff
@@ -97,13 +117,53 @@ protected ObjectFactory getObjectFactory() {
     }
 
     @Inject
+    protected JavaToolchainService getToolchainService() {
+        throw new UnsupportedOperationException();
+    }
+
+    @Inject
+    @Deprecated
     public IsolatedAntBuilder getAntBuilder() {
         throw new UnsupportedOperationException();
     }
 
+    @Inject
+    public WorkerExecutor getWorkerExecutor() {
+        throw new UnsupportedOperationException();
+    }
+    /**
+     * JavaLauncher for toolchain support
+     * @since 7.6
+     */
+    @Incubating
+    @Nested
+    public Property<JavaLauncher> getJavaLauncher() {
```

Comment:
This will need to be added to the CodeNarc.xml as a property

Reply:
Added with an empty value similar to how was done for checkstyle:
https://github.com/gradle/gradle/blob/master/subprojects/docs/src/docs/dsl/org.gradle.api.plugins.quality.CodeNarc.xml